source/includes/connection-snippets/x509-snippets/java-connection.java

```diff
@@ -0,0 +1,37 @@
+// begin x509 connection
+import java.util.Arrays;
+import org.bson.Document;
+import com.mongodb.MongoClientSettings;
+import com.mongodb.MongoCredential;
+import com.mongodb.ServerAddress;
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoIterable;
+
+public class X509Connection {
+
+    public static void main(String args[]) {
+
+        System.setProperty("javax.net.ssl.trustStore", "/etc/certs/mongodb/v3/client.truststore");
```

Comment:
Normally applications don't need their own trust store, unless they are self-signing the server certificate.  For Atlas, you shouldn't need these two lines

Reply:
Removed the two lines related to the trustStore

Reply:
This should also apply to scala.

Reply:
Removed the two lines related to the trustStore for scala as well @nathan-contino-mongo